services/reddit/subreddit-subscribers.tester.js

```diff
@@ -23,3 +23,22 @@ t.create('subreddit-subscribers (not existing subreddit)')
     label: 'reddit',
     message: 'subreddit not found',
   })
+
+t.create('subreddit-subscribers (private sub)')
+  .get('/centuryclub.json')
+  .expectBadge({
+    label: 'reddit',
+    message: 'subreddit is private',
+  })
+
+t.create('subreddit-subscribers (private sub)')
```

Comment:
could you provide some info about the purpose of this test?

Reply:
The test 'subreddit-subscribers (private sub)' is testing a subreddit that is private and will return a 403 error code, to check that it is being handled correctly. CenturyClub is a club where you have to have a specific amount of karma to get access so will always be private to shields.

Reply:
how does this differ from the live/non-mocked test immediately above? i feel like the 403 use case is already covered and am still confused by this mocked test

Reply:
I see what you mean, I was referring to a different test sorry. The mocked test returns no subscriber information, testing the part of the code that throws an error if it is missing. The live test's should in theory always return that information, or cause another error such as being private.

Reply:
Ah okay, think I have a better sense after taking a look at the service class. I think it would be better to use a different test name here that better reflects what the test is validating, perhaps:

```suggestion
t.create('subreddit-subscribers (no subscribers)')
```

I also feel like the error message that's being thrown when the `subscribers` key is missing in the response is a little confusing, especially given the API responds with a proper 404 response code for an unknown subreddit (and we use the exact same error message for both). 

I feel like the error message should be something more like `no subscribers found` as an `InvalidResponse` error instead of a `NotFound` as well

Reply:
I'm probably going to go ahead and merge, but this will need a follow up change, as at a minimum we don't want to keep two tests with the exact same name as we have right now